ui/flow.reel/flow.js

```diff
@@ -1615,34 +1611,34 @@ var Flow = exports.Flow = Component.specialize( /** @lends Flow.prototype # */ {
             }
             if (this.splinePaths.length) {
                 for (i = 0; i < length; i++) {
-                    offset = this.offset(visibleIndexes[i]);
+                    offset = this.offset(this._visibleIndexes[i]);
                     pathIndex = offset.pathIndex;
                     slideTime = offset.slideTime;
                     indexTime = this._splinePaths[pathIndex]._convertSplineTimeToBezierIndexTime(slideTime);
                     iteration = this._repetition._drawnIterations[i];
                     element = iteration.cachedFirstElement || iteration.firstElement;
                     if (indexTime !== null) {
-                        if (element.children[0]) {
+                        if (elementChildren = element.children[0]) {
```

Comment:
Because there is a conditional there. First I check if there is an element.children[0], then I can access the classList, else if I try to do directly element.children[0].classList I would get an error because children[0] might be undefined